tests/upgrade/rules/test_no_additional_args_operators.py

```diff
@@ -0,0 +1,49 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+import textwrap
+from tempfile import NamedTemporaryFile
+from unittest import TestCase
+
+import pytest
+
+from airflow.upgrade.rules.no_additional_args_in_operators import NoAdditionalArgsInOperatorsRule
+
+
+class TestNoAdditionalArgsInOperatorsRule(TestCase):
+
+    @pytest.mark.filterwarnings('always')
```

Comment:
This was needed as we suppress DeprecationWarning and PendingDeprecationWarning when running pytest on CI:

```
Starting the tests with those pytest arguments: --verbosity=0 --strict-markers --durations=100 --cov=airflow/ --cov-config=.coveragerc --cov-report=xml:/files/coverage.xml --color=yes --maxfail=50 --pythonwarnings=ignore::DeprecationWarning --pythonwarnings=ignore::PendingDeprecationWarning --junitxml=/files/test_result.xml --setup-timeout=20 --execution-timeout=60 --teardown-timeout=20 -rfEX --with-db-init tests
```

Lol took few hours for me & Dennis to figure why we were not able to replicate it when running it with breeze:

```
root@e6768532ef19:/opt/airflow# pytest tests/upgrade/rules/test_no_additional_args_operators.py
```

until we ran it with:

```
root@e6768532ef19:/opt/airflow# pytest tests/upgrade/rules/test_no_additional_args_operators.py --pythonwarnings=ignore::DeprecationWarning --pythonwarnings=ignore::PendingDeprecationWarning
```

:D

Reply:
Good work you two!